auacm/app/modules/competition_manager/views.py

```diff
@@ -222,9 +218,9 @@ def register_for_competition(cid):
             is None:
         return serve_error('Competition does not exist', response_code=404)
 
-    if current_user.admin == 1 and 'users' in request.data:
+    if current_user.admin == 1 and 'users' in request.form:
```

Comment:
I think it's actually because of the python3 switch, not Tornado. I noticed it was broken after I finished the conversion, but it seemed to be because of python3.